docs/Migration-2.3.md

```diff
@@ -22,7 +22,7 @@ Because we continue to support **minSdk 21**, **core library desugaring is requi
 
 
 ### 2.2 New helper: synchronized timestamp for `ProximityCheck`
-You can now use the static factory [ProximityCheck.withSynchronizedTime](../library/src/main/java/com/wultra/android/mtokensdk/api/operation/model/UserOperation.kt#withSynchronizedTime) to create a ProximityCheck instance with a timestamp that prefers **server-synchronized** time from PowerAuth, falling back to the device clock if synchronization is not available.
+You can now use the static factory [ProximityCheck.withSynchronizedTime](https://github.com/wultra/mtoken-sdk-android/blob/8ce306f551589c0666226af2e251f609c5c14860/library/src/main/java/com/wultra/android/mtokensdk/api/operation/model/UserOperation.kt#L253-L266) to create a ProximityCheck instance with a timestamp that prefers **server-synchronized** time from PowerAuth, falling back to the device clock if synchronization is not available.
```

Comment:
Hm, I'm wondering if this could be done better, because this commit will be deleted with the branch and could be lost in the future.

To be future-proof (so we don't have to change it each time the files have changed), I would just point to `develop` without a specific line.

Or even better, remove the link altogether; those who want can find the source code on its own...

I know we (and I) did it in the past, but it comes with a maintainance problem (keeping the link up-to-date).